src/PickerInput/hooks/useRangeValue.ts

```diff
@@ -289,11 +289,7 @@ export default function useRangeValue<ValueType extends DateType[], DateType ext
 
       // Trigger `onChange` if needed
       if (onChange && !isSameMergedDates) {
-        onChange(
-          // Return null directly if all date are empty
-          isNullValue && clone.every((val) => !val) ? null : clone,
-          getDateTexts(clone),
-        );
+        onChange(clone, getDateTexts(clone));
```

Comment:
`onChange` is used as same as `onCalendarChange` is.

https://github.com/react-component/picker/blob/8bfd695ad3e2fbc9da09614c9b79de48bf032af1/src/PickerInput/hooks/useRangeValue.ts#L152-L154